handlers/user.go

```diff
@@ -1181,6 +1181,14 @@ func UserDeletePost(w http.ResponseWriter, r *http.Request) {
 		}
 
 		Logout(w, r)
+		err = purgeAllSessionsForUser(r.Context(), user.UserID)
+		if err != nil {
+			logger.Errorf("error purging sessions for user %v: %v", user.UserID, err)
+			session.AddFlash(authInternalServerErrorFlashMsg)
```

Comment:
Shouldn't this be **SetFlash** ? (as in the error handling above)

Reply:
Both variants have been used throughout the file so I wasn't sure. `SetFlash` is calling `AddFlash` anyways, so it shouldn't matter too much, I'll apply it